WORKSPACE

```diff
@@ -43,6 +43,46 @@ container_load(
     file = "//testdata:pause.tar",
 )
 
+container_pull(
+    name = "alpine_linux_amd64",
+    registry = "index.docker.io",
+    repository = "library/alpine",
+    tag = "3.8",
+)
+
+container_pull(
+    name = "alpine_linux_armv6",
+    architecture = "arm",
+    cpu_variant = "v6",
+    os = "linux",
+    registry = "index.docker.io",
+    repository = "library/alpine",
+    tag = "3.8",
+)
+
+container_pull(
+    name = "alpine_linux_ppc64le",
+    architecture = "ppc64le",
+    registry = "index.docker.io",
+    repository = "library/alpine",
+    tag = "3.8",
+)
+
+container_pull(
+    name = "k8s_pause_amd64",
+    registry = "k8s.gcr.io",
+    repository = "pause",
```

Comment:
should we set a fixed sha here so test can be guaranteed to only fail due to resolution of the digest and not due to a change in the image being pulled?

Reply:
I changed this one to a digest pointing at the manifest list to verify that the resolved digest points at the amd64 image.

Reply:
thanks